### ISSUE TITLE
Scan crashes with non-java rules

### ISSUE THREAD

**Issue by acanda** [ybroeker/pmd-idea#18]
The PMD scan crashes when the ruleset file contains non-java rules, e.g. rules from the categories `Maven POM` or `XML`. It looks like the plugin only contains the jar for the Java rules (pmd-java.jar) and is missing those for all the other categories that come by default with PMD.

I don't know if you intentionally excluded the non-java rules. If so, then this is a bug report and I expect the plugin to just ignore the rules it doesn't support instead of crashing and not reporting any issues, even for java rules. Otherwise this is a feature request to include all rule categories that PMD supports.

This ruleset file triggers the crash:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="crash"
         xsi:noNamespaceSchemaLocation="http://pmd.sourceforge.net/ruleset_2_0_0.xsd"
         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd"
         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0">
    <description>This ruleset crashes PMD-IDEA 1.5.0</description>
    <rule ref="category/pom/errorprone.xml"/>
</ruleset>
```
The following exception is reported when the plugin tries to execute a scan:
```
java.lang.IllegalArgumentException: net.sourceforge.pmd.RuleSetNotFoundException: Can't find resource 'category/pom/errorprone.xml' for rule 'null'.  Make sure the resource is a valid file or URL and is on the CLASSPATH. Here's the current classpath: C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\bootstrap.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\util.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\jdom.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\log4j.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\jna.jar
	at net.sourceforge.pmd.RulesetsFactoryUtils.getRuleSets(RulesetsFactoryUtils.java:55)
	at net.sourceforge.pmd.processor.AbstractPMDProcessor.createRuleSets(AbstractPMDProcessor.java:82)
	at net.sourceforge.pmd.processor.AbstractPMDProcessor.processFiles(AbstractPMDProcessor.java:118)
	at com.github.ybroeker.pmdidea.pmdwrapper.PmdRunner.run(PmdRunner.java:76)
	at com.github.ybroeker.pmdidea.pmdwrapper.PmdWrapperImpl.runPmd(PmdWrapperImpl.java:46)
	at com.github.ybroeker.pmdidea.pmd.PmdAdapterDelegate.runPmd(PmdAdapterDelegate.java:30)
	at com.github.ybroeker.pmdidea.inspection.PmdInspection.runPmd(PmdInspection.java:58)
	at com.github.ybroeker.pmdidea.inspection.PmdInspection.checkFile(PmdInspection.java:44)
	at com.intellij.codeInspection.LocalInspectionTool$1.visitFile(LocalInspectionTool.java:145)
	at com.intellij.psi.impl.source.PsiJavaFileBaseImpl.accept(PsiJavaFileBaseImpl.java:473)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:65)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$7(LocalInspectionsPass.java:346)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:174)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:132)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:121)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:111)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$10(LocalInspectionsPass.java:346)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:136)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1096)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:92)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:104)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:83)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:178)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:83)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: net.sourceforge.pmd.RuleSetNotFoundException: Can't find resource 'category/pom/errorprone.xml' for rule 'null'.  Make sure the resource is a valid file or URL and is on the CLASSPATH. Here's the current classpath: C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\bootstrap.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\util.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\jdom.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\log4j.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2018.1.1\lib\jna.jar
	at net.sourceforge.pmd.RuleSetReferenceId.getInputStream(RuleSetReferenceId.java:410)
	at net.sourceforge.pmd.RuleSetFactory.parseRuleSetNode(RuleSetFactory.java:360)
	at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:233)
	at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:228)
	at net.sourceforge.pmd.RuleSetFactory.parseRuleSetReferenceNode(RuleSetFactory.java:541)
	at net.sourceforge.pmd.RuleSetFactory.parseRuleNode(RuleSetFactory.java:499)
	at net.sourceforge.pmd.RuleSetFactory.parseRuleSetNode(RuleSetFactory.java:408)
	at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:233)
	at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:228)
	at net.sourceforge.pmd.RuleSetFactory.createRuleSets(RuleSetFactory.java:183)
	at net.sourceforge.pmd.RuleSetFactory.createRuleSets(RuleSetFactory.java:164)
	at com.github.ybroeker.pmdidea.pmdwrapper.CachingRuleSetFactory.createRuleSets(CachingRuleSetFactory.java:30)
	at net.sourceforge.pmd.RulesetsFactoryUtils.getRuleSets(RulesetsFactoryUtils.java:46)
	... 34 more
```

**Comment by riquier** [comment]
I am also getting the equivalent error when trying to reference `Apex` rules
